server/task_executor_pitr_cutover.go

```diff
@@ -33,7 +34,50 @@ func (exec *PITRCutoverTaskExecutor) RunOnce(ctx context.Context, server *Server
 
 	// Currently api.TaskDatabasePITRCutoverPayload is empty, so we do not need to unmarshal from task.Payload.
 
-	return exec.pitrCutover(ctx, task, server)
+	terminated, result, err = exec.pitrCutover(ctx, task, server)
+	if err != nil {
+		return terminated, result, err
+	}
+
+	issue, err := server.store.GetIssueByPipelineID(ctx, task.PipelineID)
```

Comment:
Please re-use `getIssueByPipelineID()` in `task_executor_pitr_restore.go`.
And maybe rename it to `getNonNillIssueByPipelineID()`?

Reply:
And we could get issue once, pass it into `pitrCutover` and use it here.